From 4c12309972a6e692c946951c1cc61c41d60deb3a Mon Sep 17 00:00:00 2001
From: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Date: Thu, 12 Jun 2014 09:20:47 +0100
Subject: [PATCH 224/351] regulator: arizona-ldo1: Increase enable time

Recent evaluation of the device suggests that some systems might need an
increased enable time, increase this to be safe.

Change-Id: If3c43f8471721a869cdfd043fc1cabeeda708e5e
Signed-off-by: Charles Keepax <ckeepax@opensource.wolfsonmicro.com>
Signed-off-by: Nikesh Oswal <Nikesh.Oswal@wolfsonmicro.com>
---
 drivers/regulator/arizona-ldo1.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/regulator/arizona-ldo1.c b/drivers/regulator/arizona-ldo1.c
index 57bff05..e36a063 100644
--- a/drivers/regulator/arizona-ldo1.c
+++ b/drivers/regulator/arizona-ldo1.c
@@ -155,7 +155,7 @@ static const struct regulator_desc arizona_ldo1 = {
 	.min_uV = 900000,
 	.uV_step = 25000,
 	.n_voltages = 13,
-	.enable_time = 500,
+	.enable_time = 3000,
 
 	.owner = THIS_MODULE,
 };
-- 
1.7.9.5

